### DISCUSSION TITLE
After upgrading to Webdriverio 7 from v6 browser.config is not recognized

### DISCUSSION THREAD

**Discussion by tzurp** [webdriverio/webdriverio#6428] [Q&A] [answered]
Hi,
I am using a Typescript WebdriverIO project and upgraded it to version  7 (7.0.3). When I'm trying to compile it using tsc --build,  compilation is failed due to: Property 'config' does not exist on type 'Browser'.
Another thing after upgrade is that browser.capabilities has no properties at all thus I can't get for example the browser name: browser.capabilities.browserName (Property 'browserName' does not exist on type 'RemoteCapability'.
  Property 'browserName' does not exist on type 'W3CCapabilities').
Any ideas?

Thanks

**Comment by christian-bromann** [comment, marked as answer]
Addressed in https://github.com/webdriverio/webdriverio/pull/6429. To get the capabilities you need to:

```ts
import { Capabilities } from '@wdio/types'

(browser.capabilities as Capabilities.DesiredCapabilities).browserName
```

**Reply by tzurp** [reply]
Thanks Christian. What about: Property 'config' does not exist on type 'Browser' ? I do have browser.options but no browser.config. Is this change is by design?

**Reply by christian-bromann** [reply]
no, see #6429
